cpp/main.cc

```diff
@@ -487,6 +524,8 @@ int main(int argc, char **argv) {
   else {
     // Main loops
     TString baseDir = "/store/group/Hgg/XToYHToggbb/skimmedNanoAOD";
+    //inverted
+    TString invDir  = "/store/user/iareed/skim_Jul202023_ggbb_Data_and_DY_ABCD_mini_jobs";
```

Comment:
...Or we can use a different directory (but let's move it to the common ceph area), nstead of the suffix.